ofp/packet.go

```diff
@@ -83,6 +83,9 @@ type PacketIn struct {
 
 	// Match is used to match the packet.
 	Match Match
+
+	// Data represents the original ethernet frame received by the datapath.
+	Data bytes.Buffer
```

Comment:
For the sake of consistency with ```EchoReply``` message, it would be better set type of this field to ```[]byte```.

Reply:
Alternatively, probably, it's time to migrate ```EchoReply``` to ```bytes.Buffer```. What's your case and why do you prefer ```bytes.Buffer``` in this case?

Reply:
No particular reason for using bytes.Buffer... I changed to []byte.